webpack.config.js

```diff
@@ -1,10 +1,14 @@
 var path = require('path');
-
+var webpack = require('webpack');
 
 // THREE js replace
 //"three": "^0.74.0" -> "three": "mrdoob/three.js#35a5994828da7cebc0d8442062f784b3f9e1f818",
 //                                               #idcommit
 
+var definePlugin = new webpack.DefinePlugin({
+    __DEBUG__: JSON.stringify(JSON.parse(process.env.NODE_ENV === 'development'))
```

Comment:
Is the `JSON.parse` really needed here‽ Looks to me like `JSON.stringify` would be enough.